data-access/src/main/java/am/smartcafe/data_access/dto/req/UserRegisterRequest.java

```diff
@@ -0,0 +1,129 @@
+package am.smartcafe.data_access.dto.req;
+
+import am.smartcafe.data_access.model.Role;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import java.util.Objects;
+import java.util.Set;
+
+import lombok.Builder;
+
+@Builder
+public class UserRegisterRequest {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Size(min = 2, max = 50)
+    @NotEmpty(message = "*Please provide your first name")
+    private String firstName;
+
+    @Size(min = 2, max = 50)
+    @NotEmpty(message = "*Please provide your last name")
+    private String lastName;
+
+    @NotBlank
+    private Set<Role> role;
+
+    @Column(name = "active")
+    private int isActive;
```

Comment:
state will be set with user service

Reply:
I mean, we don't need isActive field here, do we?

Reply:
yes,of course

Reply:
changed...